{{cookiecutter.project_slug}}/tests/unit/test_package.py

```diff
@@ -25,8 +25,13 @@ def pylintrc(project_slug_dir, osparc_simcore_root_dir):
 
 {# TODO: check pytst-pylint and replace this test #}
 def test_run_pylint(pylintrc, package_dir):
-    cmd = 'pylint -j 2 --rcfile {} -v {}'.format(pylintrc, package_dir)
-    assert subprocess.check_call(cmd.split()) == 0
+    command = 'pylint -j 2 --rcfile {} -v {}'.format(pylintrc, package_dir)
+    split_command = command.split(" ")
+    pipes = subprocess.Popen(split_command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    std_out, _ = pipes.communicate()
```

Comment:
you do not import from pytest-simcore?? I thought that awas the point

Reply:
This is a very good point. I did not think this could be done!